files/main.sh

```diff
@@ -44,18 +53,4 @@ curl "<%= script.url %>" | /bin/bash
   <% end %>
 <% end %>
 
-mkdir -p /tmp/configs
```

Comment:
It should be fine for now I think, we can always add it back if needed. How this works will probably change and be simplified a bit in the near future anyway, when we look at https://trello.com/c/4xUk3FGp/105-consider-improving-how-nodes-need-to-request-and-interact-with-build-files. I'll merge this through for now :slightly_smiling_face:.